### ISSUE TITLE
[Bug] .env file created by Vemto contains DOS '\r\n' newline sequences 

### ISSUE THREAD

**Issue by jakubgg** [TiagoSilvaPereira/vemto-issues#249]
**Describe the bug**
The `.env` file created by Vemto (v. 1.0.0) contains DOS newline sequence between text blocks (`\r\n` or `0x0d 0x0a`) which causes error messages when using Laravel Sail (mainly repeated `: command not found` ) 

**To Reproduce**
1. Create new project with Vemto and generate it. 
2. In terminal in a project directory type `dos2unix -ih ./.env`
The output should show  
```
     DOS    UNIX     MAC  BOM       TXTBIN  FILE
       8      42       0  no_bom    text    ./.env
```
3. Alternatively open `.env` file in any text editor that can identify line endings and investigate lines 6,9,16,24 etc.

**Expected behavior**
Line endings for the whole file should follow the same standard (preferably the Unix style since this is a PHP project and that will be compatible with PSR-12).

**Desktop (please complete the following information):**
 - OS: MacOS 11.6
 - Version Vemto 1.0.0

**Comment by TiagoSilvaPereira** [comment]
Hi @jakubgg thank you very much for reporting

**Comment by TiagoSilvaPereira** [comment]
Fixed on 1.0.1
